package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "postinstall": "npm run install:cordova",
     "install:cordova": "cordova platform add android browser & true",
-    "postinstall:cordova": "cordova plugin add cordova-plugin-geolocation & cordova plugin add https://github.com/apache/cordova-plugin-file & cordova plugin add cordova-plugin-camera & cordova plugin add cordova-plugin-file-transfer & cordova plugin add https://github.com/hypery2k/cordova-sqlite-plugin",
+    "postinstall:cordova": "cordova plugin add cordova-plugin-geolocation & cordova plugin add https://github.com/apache/cordova-plugin-file & cordova plugin add cordova-plugin-camera & cordova plugin add cordova-plugin-file-transfer & cordova plugin add https://github.com/hypery2k/cordova-sqlite-plugin & cordova plugin add cordova-plugin-network-information & cordova plugin add cordova-plugin-device",
```

Comment:
oh man, this is really getting out of hand... I'm going to make an issue to resolve this, don't worry about it for this PR though